config/surf_snellius.py

```diff
@@ -63,7 +63,10 @@
                 {
                     'name': 'genoa',
                     'scheduler': 'slurm',
-                    'prepare_cmds': [common_eessi_init()],
+                    'prepare_cmds': [
+                        'export EESSI_SOFTWARE_SUBDIR_OVERRIDE=x86_64/amd/zen4',
```

Comment:
what does this do exactly?

Reply:
Note that it also doesn't hurt to hard code those overrides if you know what partition you are on. It simply means you're not relying on the autodetection in the init script (but I know the hardware in this partition, so that's ok :P)